src/etos_test_runner/lib/log_area.py

```diff
@@ -253,9 +278,12 @@ def __upload(
         if upload.get("auth"):
             upload["auth"] = self.__auth(**upload["auth"])
 
+        checksums = {}
         with open(log, "rb") as log_file:
+            content = log_file.read()
+            checksums["md5"] = hashlib.md5(content).hexdigest()
```

Comment:
We should probably another algorithm. Eiffel uses these: `SHA-224, SHA-256, SHA-384, SHA-512, SHA-512/224, SHA-512/256`